src/deleteIndex.ts

```diff
@@ -1,20 +1,17 @@
 import { config } from "dotenv";
-import { getPineconeClient } from "./pinecone.js";
+import { Pinecone } from "@pinecone-database/pinecone";
 import { getEnv, validateEnvironmentVariables } from "./utils/util.js";
 
 config();
+validateEnvironmentVariables();
```

Comment:
It doesn't really matter if this validation occurs inside the command function or just after the config is loaded since it's not going to change.